wordle-web/src/scripts/wordleGame.ts

```diff
@@ -10,26 +11,43 @@ export enum WordleGameStatus {
 
 export class WordleGame {
   constructor(secretWord?: string, numberOfGuesses: number = 6) {
-    if (!secretWord) secretWord = WordsService.getRandomWord()
+    //if (!secretWord) secretWord = WordsService.getRandomWord()
     this.numberOfGuesses = numberOfGuesses
-    this.restartGame(secretWord)
+    this.restartGame('defal')
   }
   guessedLetters: Letter[] = []
   guesses = new Array<Word>()
   secretWord = ''
+  numberOfSeconds = 0
   status = WordleGameStatus.Active
   guess!: Word
   numberOfGuesses = 6
+  finalTime = 0
+  finalGuesses = 0
+  useName = ''
```

Comment:
userName*

Reply:
"useName" was actually on purpose, but it wasn't a good name to choose and the variable could be removed if I fixed a few things.  I was using useName as a holder for the actual name string and "displayName" as the value returned to the user and calling the database.  I was doing that as a dumb workaround for having the name display as "Guest" when empty because I wasn't updating the display value correctly, but I should have used something like "rawName" or "nameValue" to make that more clear.